sql/2019/05_Third_Parties/05_06.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Top 100 third party domains by request volume
+SELECT
+  thirdPartyDomain,
+  COUNT(*) AS totalRequests,
+  SUM(requestBytes) AS totalBytes
+FROM (
+  SELECT
+      respSize AS requestBytes,
+      NET.HOST(url) AS requestDomain,
+      DomainsOver50Table.requestDomain as thirdPartyDomain
+    FROM
+      `httparchive.summary_requests.2019_07_01_mobile`
```

Comment:
Should this be both desktop and mobile? You can change `mobile` to `*`. This will create a special `_TABLE_SUFFIX` field that you can alias as `client` and group by.

Reply:
I think the signficance of desktop vs. mobile here is much less important than the other metrics and complicates the `LIMIT` of this query as well, so I'm inclined to just combine both without distinguishing, there was also no reason not to use `alamanac.summary_requests` here so switched to that 👍